default_gh_config.yaml

```diff
@@ -9,6 +9,7 @@ graphhopper:
   routing.ch.disabling_allowed: true
   routing.max_visited_nodes: 1500000
   graph.flag_encoders: car,bike,foot,truck
+  graph.encoded_values: road_class
```

Comment:
Any specific edge info that we use in a `custom_model_file` needs to be added here (in our case, this will very likely just be `road_class`)